### PR TITLE
config: Copy afi-safi objects from peer group to neighbor instead of …

### PR DESCRIPTION
**Pull request by qsqx**
…share the same array
https://github.com/osrg/gobgp/issues/1765

### CODE REVIEW

File: config/default.go [hunk 1]
```diff
@@ -496,7 +496,9 @@ func OverwriteNeighborConfigWithPeerGroup(c *Neighbor, pg *PeerGroup) error {
 	overwriteConfig(&c.TtlSecurity.Config, &pg.TtlSecurity.Config, "neighbor.ttl-security.config", v)
 
 	if !v.IsSet("neighbor.afi-safis") {
-		c.AfiSafis = pg.AfiSafis
+		for _, afiSafi := range pg.AfiSafis {
+			c.AfiSafis = append(c.AfiSafis, afiSafi)
+		}
```
Comment (fujita) [inline config/default.go]: Each peer has own slice but still shares State object so one peer modifies a State object then it affects the rest of peers?

Comment (qsqx) [reply config/default.go]: yes, an example is

Comment (qsqx) [reply config/default.go]: AfiSafi.State is not a pointer but a value and modification to one won't affect others.

Comment (fujita) [reply config/default.go]: oops. pushed, thanks.

**Pull request closed without merging**